KamerConnect.DataAccess.Postgres/Repositories/ChatRepository.cs

```diff
@@ -96,4 +96,102 @@ private ChatMessage ReadToChatMessage(DbDataReader reader)
             reader.GetDateTime(4)
         );
     }
+        
+    public void CreateChat(Guid chatId)
+    {
+        try
+        {
+            using (var connection = new NpgsqlConnection(_connectionString))
+            {
+                connection.Open();
+                string updateQuery = $"""
+                                      INSERT INTO chat (
+                                        chat_id, match_id)
+                                        VALUES (@chatId::uuid, @match_id::uuid
+                                      )
+                                      """;
+
+                using (var command = new NpgsqlCommand(updateQuery, connection))
+                {
+                    command.Parameters.AddWithValue("@chatId", chatId);
+                    command.Parameters.AddWithValue("@matchId", null);
+                    var result = command.ExecuteNonQuery();
+                }
+            }
+        }
+        catch (NpgsqlException e)
+        {
+            Console.WriteLine($"Error occurred while updating createChat in DB: {e.Message}");
+            throw;
+        }
+    }
+
+    public void AddPersonToChat(Guid chatId, Guid personId)
+    {
+        try
+        {
+            using (var connection = new NpgsqlConnection(_connectionString))
+            {
+                connection.Open();
+                string updateQuery = $"""
+                                      INSERT INTO person_chat (
+                                        person_id, chat_id)
+                                        VALUES (@person_id::uuid, @chat_id::uuid
+                                      )
+                                      """;
+
+                using (var command = new NpgsqlCommand(updateQuery, connection))
+                {
+                    command.Parameters.AddWithValue("@person_id", personId);
+                    command.Parameters.AddWithValue("@chatId", chatId);
+                    var result = command.ExecuteNonQuery();
+                }
+            }
+        }
+        catch (NpgsqlException e)
+        {
+            Console.WriteLine($"Error occurred while adding person to chat in DB: {e.Message}");
+            throw;
+        }
+    }
+
+    public void CreateChatWithPersons(List<Guid> personIds)
```

Comment:
Gewoon rename naar create het feit dat het de chatRepository is zegt dan al dat het een chat is